examples/apps/hello_world/index.js

```diff
@@ -1,10 +1,14 @@
 var alexa = require('alexa-app');
+var pkg = require('./package.json');
+var applicationId = (typeof pkg.alexa == 'object' ? pkg.alexa.applicationId : "");
```

Comment:
Supposedly we know exactly what is in our package.json, doesn't seem like the condition is necessary?